components/Memory.js

```diff
@@ -31,15 +33,25 @@ const Memory = (props) => {
         }
     }
 
+    useEffect(async () => {

```

Comment:
Looked at this again and apparently you can't just do async on useEffect which I forgot: https://www.robinwieruch.de/react-hooks-fetch-data

You just need to create an sync function and then run it. I think we need to do the same for the other useEffect

Reply:
I think we also add in a try catch for the searchSpotifyTracks just to get better logging when there is an error for now

Reply:
Cool, fixed!